aiida/cmdline/utils/pluginable.py

```diff
@@ -22,20 +22,36 @@ class Pluginable(click.Group):
 
     def __init__(self, *args, **kwargs):
         """Initialize with entry point group."""
+        self._isolated_mode = False
```

Comment:
Perhaps add a comment like
`# see set_isolated_mode() for what this variable does`